sim/mcuboot-sys/src/c.rs

```diff
@@ -66,6 +69,8 @@ impl BootGoResult {
 pub fn boot_go(multiflash: &mut SimMultiFlash, areadesc: &AreaDesc,
                counter: Option<&mut i32>, image_index: Option<i32>,
                catch_asserts: bool) -> BootGoResult {
+    init_psa();
```

Comment:
My first idea was that it would be better the introduce a crypto backend agnostic initialization function here and invoke the required functions based on the sim configuration (in this case psa-crypto-api). But then I had a look at the implementation of `invoke_boot_go()` and now I think that the crypto initialization step is better suited there.
Would it be possible or it needs to be here to ensure `psa_crypto_init()` and `mbedtls_test_enable_insecure_external_rng()` are called once?

Reply:
I believe the two of them do need to be called at once. I seem to remember there being a strange behaviour which requires `psa_init()` to be called regardless of if it is used or not (which is why there was a stubbed version of the function); if it was removed from here then adding the option of psa-crypto-api was not possible.

Reply:
I would suggest to name this function `init_crypto` or `crypto_init` instead, because it has nothing to do with PSA here - only the implementation of the function calls `psa_crypto_init` if `feature = "psa-crypto-api"`